src/modules/peek/Peek.FilePreviewer/Peek.FilePreviewer.csproj

```diff
@@ -103,12 +103,6 @@
     </Page>
   </ItemGroup>
 
-  <ItemGroup>
```

Comment:
Seems like unrelated changes here. Is this needed?

Reply:
This was me removing a duplicate entry I found in the project file, and it's not related to the memory leak issue. I can split it out into a separate PR if you'd prefer.